### PR TITLE
Referral program ui

### PR DESCRIPTION
**Pull request by akorunska**
ONXP-1134 Create UI for referral program functionality
Implemented UI for referral program

### CODE REVIEW

**Comment by goonerDp** [comment]
After modal is opened
image

**Comment by goonerDp** [comment]
To much space
image

**Comment by goonerDp** [comment]
Tables are not responsive.
image

File: src/components/paginated-list/ExchangeHistory.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const exchangeHistoryColumns = [
 function ExchangeHistory(props) {
 	return (
 		<>
-			<TransactionsTable
+			<PaginationTable
 				columns={exchangeHistoryColumns}
 				rowKey={"trxHast"}
```
Comment (goonerDp) [inline src/components/paginated-list/ExchangeHistory.js]: rowKey={"trxHast"} should be rowKey=**"trxHast"**

File: src/components/paginated-list/PaginationTable.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import { Table } from "antd";
 
-class TransactionsTable extends Component {
+class PaginationTable extends Component {
```
Comment (goonerDp) [inline src/components/paginated-list/PaginationTable.js]: "PaginatedTable" ?

File: src/components/paginated-list/ReferralsList.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		render: addr => (addr ? addr : "n/a"),
+	},
+];
+
+function ReferralsList(props) {
+	return (
+		<>
+			<PaginationTable
+				columns={referralsTableColumns}
+				rowKey={"addr"}
```
Comment (goonerDp) [inline src/components/paginated-list/ReferralsList.js]: If react.js prop is a string it should be passed without curly braces.
rowKey="addr"

**Comment by akorunska** [comment]
> Tables are not responsive. Please, take a look at how I solved this problem for trx list in master.
> image

I used the same trx list component for this tables(now renamed to PaginatedTable), so after merge with master same solution is applied to referrals and rewarded transactions lists.

**Pull request merged**